app/assets/stylesheets/default_theme/table.scss

```diff
@@ -47,7 +47,8 @@
     }
   }
 
-  td {
+  td,
+  > tbody > tr > td {
```

Comment:
Ah I see. :+1: 

Can you maybe add a comment to explain that in the source? It's not obviously by just looking at it.